mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -586,6 +589,31 @@ Use the `google_identity_platform_config` resource instead. It contains a more c
 
 `reconcile_connections` previously defaults to true. Now it will default from the API.
 
+
+## Resource: `google_dataflow_flex_template_job`
```

Comment:
adding @rileykarson to LGTM this section

Reply:
LGTM (gonna leave clicking the green check to you)